src/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@
 # SOFTWARE.
 ##
 
+SET(RMLUI_SKIP_INSTALL ON)
```

Comment:
why did you move this here? It should go before the rmlui code is included, in the file where you commented it out.

Reply:
I was unsure whether to keep it there or move it, thanks for the clarification.

Reply:
In CMake, once you've created a "target", the only way to modify that target is through directly setting it's "target properties"

So here, setting a "loose" variable like this has no effect on anything.

That's why i said that you should be setting this variable in the 3rdparty/CMakeLists.txt file.